packages/block-editor/src/text-input/index.js

```diff
@@ -27,6 +28,7 @@ const settings = {
 	],
 	icon: <TextInputIcon />,
 	edit: Edit,
+	variations: variation,
```

Comment:
Nit: you could also import the variable as `variations` ad just put `variations,` in here making it slightly shorter.